SplunkRumWorkspace/SplunkRum/SplunkRumTests/ConnectionInstrumentationTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import Foundation
+import XCTest
+import Atomics
+class ConnectionInstrumentationTests: XCTestCase {
+    func testConnection() throws {
+        try initializeTestEnvironment()
+        var req = URLRequest(url: URL(string: "http://127.0.0.1:8989/data")!)
+        req.httpMethod = "GET"
+        NSURLConnection.sendAsynchronousRequest(req, queue: OperationQueue.main) {(_, _, _) in
+           print("Got data..")
```

Comment:
Can we test here that the callback was called?

Reply:
I checked it with sendSynchronousRequest method and now test succed.

Reply:
sendAsynchronousRequest has no return parameter, so that callback is not return to it. Because of that local span was not created and test failed. 
I Implemented test case using sendSynchronousRequest method , test succeed.
I commit same.